### PR TITLE
Updated README.md 

### PR DESCRIPTION
**Pull request by ramshaashraf**
Heyy, I have updated the README.md file, let me know how you like it or if any other changes should be made. Thank you :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,69 +1,41 @@
...
 ### Technologies used:-
 1. Front-End Development:
 - Html 5.
-- Css.
+- CSS.
 - Javascript.
-- <a href="https://theonlinebookstore.herokuapp.com" target="_blank">Click Here for Front End View</a>
 
-2. Back-End Development
-- Java [JDK 8+]
```
Comment (shashirajraja) [inline README.md:42]: version specification is required

File: README.md [hunk 2]
```diff
@@ -1,69 +1,41 @@
...
+### ================= Dummy Database Initialization =================
 
 STEP 1: Open MySQL Command Prompt or MySQL Workbench
 
 STEP 2: Login to the administrator user as : ```mysql -u <username> -p``` (Enter Password if asked)
 
-STEP 3 :Copy paste the following MySql Commands:
+STEP 3: Copy paste the following MySql Commands-
 ```MySQL
 create database if not exists onlinebookstore;
```
Comment (shashirajraja) [inline README.md:41]: The database query seems to be outdated with current command in readme. Please check it once

**Comment by ramshaashraf** [comment]
hey, i have made the necessary changes. i'm not too sure about the sql commands- please check it once

**Review by shashirajraja — APPROVED** [review]
Thanks for your contribution @ramshaashraf

**Pull request merged**